README.rst

```diff
@@ -14,24 +35,24 @@ Setting Up the Demo
 
 Copy (clone) this structural-imbalance-demo repository to your local machine.
 
-To set up the required dependencies, in the root directory of a copy (clone) of this repository, run the following:
+In the root directory of your copy of this repository, execute one of the following two commands:
 
-.. code-block:: bash
+A. Install Dependencies for Local CPU Execution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  pip install .
+.. code-block:: bash
 
-Configuring the Demo
---------------------
+   pip install .
 
-To run the demo on the QPU, extra dependencies must be installed by running the following:
+B. Install Dependencies for D-Wave System Execution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code-block:: bash
 
-  pip install .[qpu] --extra-index-url https://pypi.dwavesys.com/simple
+   pip install .[qpu] --extra-index-url https://pypi.dwavesys.com/simple
 
-Access to a D-Wave system must be configured, as described in the `dwave-cloud-client
-<http://dwave-cloud-client.readthedocs.io/en/latest/reference/intro.html#configuration>`_ documentation. A default
-solver is required.
+Access to a D-Wave system must be configured, as described in the
+`dwave-cloud-client<http://dwave-cloud-client.readthedocs.io/en/latest/reference/intro.html#configuration>`_ documentation. A default solver is required.
```

Comment:
This link is not displaying properly. I think it needs a space between the label and the <> enclosed hyperlink.

Reply:
Fixed